Source/SwiftLintFramework/Rules/NimbleOperatorRule.swift

```diff
@@ -60,4 +71,18 @@ public struct NimbleOperatorRule: ConfigurationProviderRule, OptInRule {
                 location: Location(file: file, characterOffset: $0.0.location))
         }
     }
+    public func correctFile(_ file: File) -> [Correction] {
+        let variable = "\\s*(.*?)\\s*"
+        let patterns = [
+            "expect\\(\(variable)\\)\\.to\\(equal\\(\(variable)\\)\\)": "expect($1) == $2",
```

Comment:
we should keep the list of supported operators in one place, since we also use them in `validateFile(_:)`.